AWS.md

```diff
@@ -1,7 +1,18 @@
 General
 -------
 
- * Use CloudFormation or ElasticBeanStalk to define infrastructure in AWS.
+ * Use [CDK](https://github.com/guardian/cdk) CloudFormation or ElasticBeanStalk to define infrastructure in AWS
```

Comment:
I find this first line a bit confusing. How about (something like):

> * Provision and manage all AWS resources using infrastructure as code
>   * Prefer to use [CDK](https://github.com/guardian/cdk) to generate CloudFormation where possible. You might find that older projects still use CloudFormation directly, these should be migrated to CDK where possible.

I also think we can remove any reference to `ElasticBeanStalk` here?

Reply:
Yes, that's much better! Done.